.github/workflows/update-toast.yml

```diff
@@ -28,15 +28,15 @@ jobs:
     - name: Update conf.py
       if: steps.latest_release.outputs.version != env.CURRENT_VERSION
       env:
-        CURRENT_VERSION: v2.8.2
+        CURRENT_VERSION: v2.8.2v2.8.2
       run: |
         sed -i "s/Litestar $CURRENT_VERSION/Litestar ${{ steps.latest_release.outputs.version }}/g" conf.py
         sed -i "s|/release-notes/changelog.html#$CURRENT_VERSION|/release-notes/changelog.html#${{ steps.latest_release.outputs.version }}|g" conf.py
 
     - name: Update workflow file
       if: steps.latest_release.outputs.version != env.CURRENT_VERSION
       run: |
-        sed -i "s/CURRENT_VERSION: $CURRENT_VERSION/CURRENT_VERSION: ${{ steps.latest_release.outputs.version }}/g" .github/workflows/update-toast.yml
+        sed -i "s/CURRENT_VERSION: v2.8.2$CURRENT_VERSION/CURRENT_VERSION: v2.8.2${{ steps.latest_release.outputs.version }}/g" .github/workflows/update-toast.yml
```

Comment:
suppose its not meant to be doing this either?

Reply:
yeah.. im no sed guru..